boost-gradle/src/main/groovy/io/openliberty/boost/gradle/tasks/BoostPackageTask.groovy

```diff
@@ -104,7 +122,7 @@ public class BoostPackageTask extends AbstractBoostTask {
             //The task will perform this before any other task actions
             doFirst {
 
-                libertyServerPath = "${project.buildDir}/wlp/usr/servers/BoostServer"
+                libertyServerPath = "${project.buildDir}/wlp/usr/servers/${project.liberty.server.name}"
                 
```

Comment:
We don't have a Maven equivalent for this name, right?  Is this a consequence of the fact that the user configures liberty-gradle directly whereas LMP is wrapped?

Reply:
Right, we can just get the properties from liberty-gradle in this case.